EternalModLoader/EternalModLoader.cs

```diff
@@ -321,13 +321,29 @@ public static void ReplaceChunks(MemoryStream memoryStream, ResourceInfo resourc
 
                         if (res != 0)
                         {
+                            Console.ForegroundColor = ConsoleColor.Red;
+                            Console.Write("ERROR: ");
+                            Console.ResetColor();
+                            Console.WriteLine($"Failed to parse {resourceInfo.Name}/{mod.Name}");
                             continue;
                         }
 
                         BlangFile blangFile;
                         using (var blangMemoryStream = new MemoryStream(blangFileBytes))
                         {
-                            blangFile = BlangFile.ParseFromMemory(blangMemoryStream);
+                            try
+                            {
+                                blangFile = BlangFile.ParseFromMemory(blangMemoryStream);
+                            }
+                            catch
+                            {
+                                Console.ForegroundColor = ConsoleColor.Red;
+                                Console.Write("ERROR: ");
+                                Console.ResetColor();
+                                Console.WriteLine($"Failed to parse {resourceInfo.Name}/{mod.Name} - are you trying to inject to the wrong .resources?");
```

Comment:
_Extremely_ nitpicky, but...?
```suggestion
                                Console.WriteLine($"Failed to parse {resourceInfo.Name}/{mod.Name} - are you trying to change strings in the wrong .resources archive?");
```

Reply:
Committed, thanks for the suggestion!